apps/client/src/features/rundown/rundown-header/RundownHeader.tsx

```diff
@@ -22,6 +29,22 @@ export default function RundownHeader() {
           Edit
         </Button>
       </ButtonGroup>
+      <ButtonGroup isAttached>
+        <Button
+          size='sm'
+          variant={offsetMode === OffsetMode.Absolute ? 'ontime-filled' : 'ontime-subtle'}
+          onClick={setAbsoluteOffset}
```

Comment:
if this continues being slow, how about using `onMouseDown`? that could help it feel more responsive